ui/client/components/graph/nodeDetailsModal.js

```diff
@@ -35,13 +35,17 @@ class NodeDetailsModal extends React.Component {
   }
 
   componentWillReceiveProps(props) {
+    const isChrome = !!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime);
+
     const newState = {
       editedNode: props.nodeToDisplay,
       currentNodeId: props.nodeToDisplay.id,
       subprocessContent: null
     }
+
     //TODO more smooth subprocess loading in UI
-    if (props.nodeToDisplay && props.showNodeDetailsModal && (NodeUtils.nodeType(props.nodeToDisplay) === "SubprocessInput")) {
+    //Subprocess work only on chrome, there is problem with jonint and SVG
+    if (isChrome && props.nodeToDisplay && props.showNodeDetailsModal && (NodeUtils.nodeType(props.nodeToDisplay) === "SubprocessInput")) {
```

Comment:
please log some warn in console in `!isChrome`

Reply:
Changed